FalmouthFalcons/iteration-3
# Jim category list

Description:
Added links to navbar, and changed image to show only if one exits

Number of Fixes:
Multiple

Related Ticket(s):
#7 
Problem to Solve:
Photo did show missing image if one didn't exits. Problem corrected

Proposed Changes:
N/A

Expected Behavior:
Styling and image missing no longer present

Steps to Test Solution
1) git pull origin master
2) git fetch
3) git pull jim-category-list

Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

Deploy Notes:
- Notes regarding deployment the contained body of work.
- These should note any db migrations, etc.
